samples/JwtBearerSample/Startup.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Newtonsoft.Json.Linq;
+
+namespace JwtBearerSample
+{
+    public class Startup
+    {
+        // Shared between users in memory
+        public IList<Todo> Todos { get; } = new List<Todo>();
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit http://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddAuthentication();
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app)
+        {
+            // Simple error page to avoid a repo dependency.
+            app.Use(async (context, next) =>
+            {
+                try
+                {
+                    await next();
+                }
+                catch (Exception ex)
+                {
+                    if (context.Response.HasStarted)
+                    {
+                        throw;
+                    }
+                    context.Response.StatusCode = 500;
+                    await context.Response.WriteAsync(ex.ToString());
+                }
+            });
+
+            app.UseIISPlatformHandler();
+
+            app.UseDefaultFiles();
+            app.UseStaticFiles();
+
+            app.UseJwtBearerAuthentication(options =>
+            {
+                options.AutomaticAuthenticate = true;
+                options.AutomaticChallenge = true;
+                // You also need to update /wwwroot/app/scripts/app.js
+                options.Authority = "https://login.windows.net/tratcheroutlook.onmicrosoft.com";
+                options.Audience = "63a87a83-64b9-4ac1-b2c5-092126f8474f";
+            });
+
+            // [Authorize] would usually handle this
+            app.Use(async (context, next) =>
+            {
+                // var user = await context.Authentication.AuthenticateAsync(JwtBearerDefaults.AuthenticationScheme);
```

Comment:
I see that it's explained in the end-of-line comment below. I think I would clarify by moving that comment up and explaining this is what you do if `options.AutomaticAuthenticate` isn't true.